package ClassWork.Lessons_20_per_29.L27_ExceptionsErrors_2.P1_Tests;

public class T3 {
    public static void main(String[] args) throws Exception {
        Exception exception = null;
        throw exception;
    }
}
